src/contrib/cluster/Akka.DistributedData/Serialization/Proto/OtherMessage.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Akka.DistributedData.Serialization.Proto.Msg
+{
+    internal sealed partial class OtherMessage : IComparable<OtherMessage>
```

Comment:
Needed to allow `OtherMessage` can be sorted

Reply:
Would it be easier to just write an [`IComparer`](https://docs.microsoft.com/en-us/troubleshoot/dotnet/csharp/use-icomparable-icomparer)? So we don't have to hack around a generated class?